### DISCUSSION TITLE
FollowUpBoss

### DISCUSSION THREAD

**Discussion by james-luther** [aquasecurity/trivy#5324] [Adopters]
### Please select all the scan targets that you are using

- [ ] Container Images
- [X] Filesystem
- [X] Git Repository
- [X] Virtual Machine Images
- [ ] Kubernetes
- [X] AWS

### Which scanners are you using on those scan targets?

- [X] OS packages and software dependencies in use (SBOM)
- [X] Known vulnerabilities (CVEs)
- [X] IaC issues and misconfigurations
- [X] Sensitive information and secrets
- [X] Software licenses

### Additional Information

...
At FollowUpBoss we have Trivy integrated within our CircleCI pipeline to perform security scanning and reporting to Defectdojo and Dependencytrack. These two tools, along with Trivy, help us to manage vulnerabilities, software licenses, and misconfiguration, as well as perform software supply chain management.

**Comment by AnaisUrlichs** [comment]
Great! Thank you so much for taking the time to share 😊
